### DISCUSSION TITLE
重新安装app后 专业版变成了免费版

### DISCUSSION THREAD

**Discussion by tigersZ** [getlantern/lantern-forum-cn#1015] [Q&A] [answered]
账号：zuo9tigers@hotmail.com
支付凭证：https://img.bian666.cf/file/1730521063680_1000013713.jpg
请协助

**Comment by MaxMKMax** [comment]
@tigersZ 您好. 已经帮您提交问题上去了. 谢谢您的耐心等待.

**Reply by tigersZ** [reply]
https://img.bian666.cf/file/1731595063451_1000050999.jpg

我尝试使用连接码 但总是提示存在技术问题

**Comment by MaxMKMax** [comment]
@tigersZ 您好. 我再重新帮您提交问题多一次

**Comment by greySkiesMarsh** [comment, marked as answer]
Hi @tigersZ 

I can see that your account is current in our system, but no devices have been authorized. Please use the following code and the email you submitted above to log in with a complimentary month of Lantern Pro.

HFQHH-F4DCG-D9CKR-6CXHF-WG829

**Reply by tigersZ** [reply]
thanks for help, the problem had been sovled
